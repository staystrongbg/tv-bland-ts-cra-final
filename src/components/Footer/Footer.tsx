import styles from '../../styles/Footer.module.scss';
const Footer: React.FC = () => {
  return (
    <footer className={styles.footer}>
      <p>TV Bland</p>
      <p>Copyright TV Bland&copy;2022</p>
    </footer>
  );
};

export default Footer;
