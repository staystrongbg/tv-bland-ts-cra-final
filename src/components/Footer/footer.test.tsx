import React from 'react';
import renderer from 'react-test-renderer';
import Footer from './Footer';

it('render app', () => {
  const render = renderer.create(<Footer />).toJSON();
  expect(render).toMatchSnapshot();
});
