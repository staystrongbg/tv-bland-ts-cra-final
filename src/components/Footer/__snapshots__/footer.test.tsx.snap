// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render app 1`] = `
<footer
  className="footer"
>
  <p>
    TV Bland
  </p>
  <p>
    Copyright TV BlandÂ©2022
  </p>
</footer>
`;
